holo-isis/src/packet/tlv.rs

```diff
@@ -268,9 +268,21 @@ pub struct Ipv6Reach {
 )]
 #[derive(Deserialize, Serialize)]
 pub struct Ipv6ReachSubTlvs {
+    pub bier: Vec<BierInfoTlv>,
     pub unknown: Vec<UnknownTlv>,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+#[derive(new)]
+#[derive(Deserialize, Serialize)]
+pub struct BierInfoTlv {
```

Comment:
Please rename to `BierInfoSubTlv` and move it to `holo-isis/src/packet/subtlvs/prefix/mod.rs`. The idea is to keep Sub-TLV code separate to improve organization.

Reply:
Fixed in [3568865](https://github.com/holo-routing/holo/pull/48/commits/35688658bee27b022a4e0c51fe269ab7dd6de8f8).